services/121-service/src/payments/payments.service.ts

```diff
@@ -284,6 +277,10 @@ export class PaymentsService {
     );
 
     if (!dryRun && referenceIds.length > 0) {
+      await this.checkFspConfigurationsOrThrow(
```

Comment:
I'm not sure this is ideal. If I'm creating a payment, I probably want to get the error as early as possible.